nebullvm/operations/optimizations/utils.py

```diff
@@ -1,9 +1,13 @@
-from typing import List, Callable
-
+from typing import Callable, List, Union
```

Comment:
pathlib and typing should be in the first block and sorted ;)
nebullvm import should be in the second block